packages/console/src/onboarding/constants/index.ts

```diff
@@ -14,5 +14,5 @@ export const aboutCloudPreviewLink = 'https://docs.logto.io/about/cloud-preview?
 
 export const gtagAwTrackingId = 'AW-11124811245';
 /** This ID indicates a user has truly signed up for Logto Cloud. */
-export const gtagSignUpConversionId = `${gtagAwTrackingId}/RVejCKC65qMYEO3L3Lgp`;
+export const gtagSignUpConversionId = `AW-11192640559/ZuqUCLvNpasYEK_IiNkp`;
```

Comment:
Is it OK to expose the id to the public?

Reply:
very ok